client/service/src/client/wasm_override.rs

```diff
@@ -82,6 +83,30 @@ impl FetchRuntimeCode for WasmBlob {
 	}
 }
 
+
+#[derive(Debug, thiserror::Error)]
+#[allow(missing_docs)]
+pub enum WasmOverrideError {
+	#[error("Failed to get runtime version: {0}")]
+	VersionInvalid(String),
+
+	#[error("WASM override IO error")]
+	Io(PathBuf, #[source] std::io::Error),
+
+	#[error("Overwriting WASM requires a directory where local \
+	WASM is stored. {} is not a directory", .0.display())]
+	NotADirectory(PathBuf),
+
+	#[error("Duplicate WASM Runtimes found: \n{}\n", .0.join("\n") )]
+	DuplicateRuntime(Vec<String>),
+}
+
+impl Into<sp_blockchain::Error> for WasmOverrideError {
+	fn into(self) -> sp_blockchain::Error {
+		sp_blockchain::Error::Application(Box::new(self))
+	}
+}
```

Comment:
```suggestion
impl From<WasmOverrideError> for sp_blockchain::Error {
	fn from(err: WasmOverrideError) -> Self {
		Self::Application(Box::new(err))
	}
}
```

It is always recommend to implement `From`.